front/lib/api/assistant/configuration.ts

```diff
@@ -1405,6 +1405,19 @@ async function _createAgentDataSourcesConfigData(
     processConfigurationId: ModelId | null;
   }
 ): Promise<AgentDataSourceConfiguration[]> {
+  // Although we have the capability to support multiple workspaces,
+  // currently, we only support one workspace, which is the one the user is in.
+  // This allows us to use the current authenticator to fetch resources.
+  const allWorkspaceIds = _.uniqBy(dataSourceConfigurations, "workspaceSId");
```

Comment:
the property is `workspaceId` 
 maybe  ```new Set(dataSourceConfigurations.map(ds => ds.workspaceId)``` would be more efficient than lodash ?

Reply:
Fair ... fixed.